images/stradnieks.svg

```diff
@@ -0,0 +1,27 @@
+<svg width="542" height="480" viewBox="0 0 542 480" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Piedāvāju images mapi likt iekš static/images. Jo saturs nemainīsies, lai visi .css, .js un attēli atrodas iekš static.
Attiecīgi jāpārsauc būs visur tālāk kodā šis te ceļš līdz bildēm.